internal/testutil/unit/http_mocker.go

```diff
@@ -28,14 +28,36 @@ type MockHTTPDataConfig struct {
 	IsDiffMustSubstrings []string
 	QueryVars            []string
 	AllowMissingRequests bool
+	AllowOutOfOrder      bool
+}
+
+func (c MockHTTPDataConfig) WithAllowOutOfOrder() MockHTTPDataConfig { //nolint: gocritic // Want each test run to have its own config (hugeParam: c is heavy (112 bytes); consider passing it by pointer)
+	c.AllowOutOfOrder = true
+	return c
 }
 
 func IsCapture() bool {
-	return slices.Contains([]string{"yes", "1", "true"}, strings.ToLower(os.Getenv(EnvNameHTTPMockerCapture)))
+	val, err := strconv.ParseBool(os.Getenv(EnvNameHTTPMockerCapture))
+	if err != nil {
+		return false
+	}
+	return val
```

Comment:
```suggestion
	val, _ := strconv.ParseBool(os.Getenv(EnvNameHTTPMockerCapture))
	return val
```

false is the default value is something goes wrong, no need to do that if logic, same for all funcs

Reply:
Thanks. Changed in 21721295703853641c79d737eaee871af8c23a4e